### PR TITLE
updated article flow to match initial table

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/661

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: migration/MySites-to-OneDrive-Migration-Guide.md [hunk 1]
```diff
@@ -88,6 +78,19 @@ Your environment will not be impacted while SMAT performs a scan of your environ
 
 
 
+## Prepare your OneDrive environment
+Before migrating your My Sites content, you must pre-provision your users in OneDrive: 
+
+1. [Prepare to provision users through directory synchronization to Office 365](https://support.office.com/en-us/article/prepare-to-provision-users-through-directory-synchronization-to-office-365-01920974-9e6f-4331-a370-13aea4e82b3e).  Provisioning users with directory synchronization requires more planning and preparation than simply managing your work or school account directly in Office 365. The additional planning and preparation tasks are required to ensure that your on-premises Active Directory synchronizes properly to Azure Active Directory.
```
Comment (nenonix) [inline migration/MySites-to-OneDrive-Migration-Guide.md]:
```suggestion
1. [Prepare to provision users through directory synchronization to Office 365](https://support.office.com/article/prepare-to-provision-users-through-directory-synchronization-to-office-365-01920974-9e6f-4331-a370-13aea4e82b3e).  Provisioning users with directory synchronization requires more planning and preparation than simply managing your work or school account directly in Office 365. The additional planning and preparation tasks are required to ensure that your on-premises Active Directory synchronizes properly to Azure Active Directory. 
```

File: migration/MySites-to-OneDrive-Migration-Guide.md [hunk 2]
```diff
@@ -88,6 +78,19 @@ Your environment will not be impacted while SMAT performs a scan of your environ
 
 
 
+## Prepare your OneDrive environment
+Before migrating your My Sites content, you must pre-provision your users in OneDrive: 
+
+1. [Prepare to provision users through directory synchronization to Office 365](https://support.office.com/en-us/article/prepare-to-provision-users-through-directory-synchronization-to-office-365-01920974-9e6f-4331-a370-13aea4e82b3e).  Provisioning users with directory synchronization requires more planning and preparation than simply managing your work or school account directly in Office 365. The additional planning and preparation tasks are required to ensure that your on-premises Active Directory synchronizes properly to Azure Active Directory. 
+
+2. [Pre-provision OneDrive for users in your organization](https://support.office.com/en-us/article/Pre-provision-OneDrive-for-users-in-your-organization-ceef6623-f54f-404d-8ee3-3ce1e338db07).
```
Comment (nenonix) [inline migration/MySites-to-OneDrive-Migration-Guide.md]:
```suggestion
2. [Pre-provision OneDrive for users in your organization](https://support.office.com/article/Pre-provision-OneDrive-for-users-in-your-organization-ceef6623-f54f-404d-8ee3-3ce1e338db07).  
```

File: migration/MySites-to-OneDrive-Migration-Guide.md [hunk 3]
```diff
@@ -88,6 +78,19 @@ Your environment will not be impacted while SMAT performs a scan of your environ
...
+
+3. [Configure Office 365 for SharePoint Hybrid](https://docs.microsoft.com/en-us/sharepoint/hybrid/configure-office-365-for-sharepoint-hybrid) (optional). With SharePoint Server hybrid, productivity services in SharePoint Online can be integrated with on-premises SharePoint Server to provide unified functionality and access to data. For enterprises that want to gradually move their existing on-premises SharePoint Server services to the cloud, SharePoint Server hybrid provides a staged migration path by extending high-impact SharePoint Server workloads to SharePoint Online. </br></br>A SharePoint Server hybrid environment enables trusted communications between SharePoint Online and SharePoint Server. When you have established this trust framework, you can configure integrated functionality between services and features such as Search, Follow, and user profiles. You will need to set up some basic integration between Office 365 for enterprises and SharePoint Server before you can configure a hybrid environment.
```
Comment (nenonix) [inline migration/MySites-to-OneDrive-Migration-Guide.md]:
```suggestion
3. [Configure Office 365 for SharePoint Hybrid](https://docs.microsoft.com/sharepoint/hybrid/configure-office-365-for-sharepoint-hybrid) (optional). With SharePoint Server hybrid, productivity services in SharePoint Online can be integrated with on-premises SharePoint Server to provide unified functionality and access to data. For enterprises that want to gradually move their existing on-premises SharePoint Server services to the cloud, SharePoint Server hybrid provides a staged migration path by extending high-impact SharePoint Server workloads to SharePoint Online. 

A SharePoint Server hybrid environment enables trusted communications between SharePoint Online and SharePoint Server. When you have established this trust framework, you can configure integrated functionality between services and features such as Search, Follow, and user profiles. You will need to set up some basic integration between Office 365 for enterprises and SharePoint Server before you can configure a hybrid environment.
```

**Pull request merged**